### PR TITLE
{bio}[GCC/11.2.0] GlimmerHMM v3.0.4c

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/73b122d8e8f3cad75741df055f1a1ae6 for a full test report.

File: easybuild/easyconfigs/g/GlimmerHMM/GlimmerHMM-3.0.4c-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# make sure -O0 is not used as compiler option
+prebuildopts = "ls makefile train/makefile | xargs sed -i 's/-O0 .*//g' && "
+
+# also build in 'train' subdirectory to overwrite pre-compiled binaries
+buildopts = "&& cd ../train && make"
+
+local_train_files = ['build1', 'build2', 'build-icm', 'build-icm-noframe', 'erfapp', 'falsecomp',
+                     'findsites', 'karlin', 'score', 'score2', 'scoreATG', 'scoreATG2', 'scoreSTOP',
+                     'scoreSTOP2', 'splicescore', 'trainGlimmerHMM']
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GlimmerHMM/GlimmerHMM-3.0.4c-GCC-11.2.0.eb:31]: `trainGlimmerHMM`:
* Perl script, so there should be a Perl dependency
* `#!/usr/bin/perl`
* Requires the five `.pm` files from the same `train` directory

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/g/GlimmerHMM/GlimmerHMM-3.0.4c-GCC-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GlimmerHMM/GlimmerHMM-3.0.4c-GCC-11.2.0.eb:35]: fixed in 5e13c679fe

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/6fd763287957b4195f5ee699279fcc9a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/20947718c40a254e6413dd5b02f567c5 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**